lib/modules/home/screens/search.dart

```diff
@@ -34,7 +33,7 @@ class _SearchScreenState extends State<SearchScreen> {
   late AlgoliaQuery query;
   var timer = Timer(Duration.zero, () {});
 
-  var sorts = [
+  var sorts = const [
```

Comment:
Why is it `var sorts = const ...` and not `const sorts = ...`? Will you ever need to reassign or modify `sorts`?

Reply:
It can also be `static` since it doesn't depend on the instance